bundles/org.openhab.binding.mielecloud/README.md

```diff
@@ -0,0 +1,648 @@
+| drying_target_raw | Number | The raw target drying step of the laundry. | Yes |
+| pre_heat_finished | Switch | Indicates whether the pre-heating finished. | Yes |
+| temperature_target | Number | The target temperature of the device. | Yes |
+| temperature_current | Number | The currently measured temperature of the device. | Yes |
+| ventilation_power | String | The current ventilation power of the hood. | Yes |
+| ventilation_power_raw | Number | The current raw ventilation power of the hood. | Yes |
+| error_state | Switch | Indication flag which signals an error state for the device. | Yes |
+| info_state | Switch | Indication flag which signals an information of the device. | Yes |
+| fridge_super_cool | Switch | Start the super cooling mode of the fridge. | No |
+| freezer_super_freeze | Switch | Start the super freezing mode of the freezer. | No |
+| super_cool_can_be_controlled | Switch | Indicates if super cooling can be toggled. | Yes |
+| super_freeze_can_be_controlled | Switch | Indicates if super freezing can be toggled | Yes |
+| fridge_temperature_target | Number | The target temperature of the fridge. | Yes |
+| fridge_temperature_current | Number | The currently measured temperature of the fridge. | Yes |
+| freezer_temperature_target | Number | The target temperature of the freezer. | Yes |
+| freezer_temperature_current | Number | The currently measured temperature of the freezer. | Yes |
+| top_temperature_target | Number | The target temperature of the top area. | Yes |
+| top_temperature_current | Number | The currently measured temperature of the top area. | Yes |
+| middle_temperature_target | Number | The target temperature of the middle area. | Yes |
+| middle_temperature_current | Number | The currently measured temperature of the middle area. | Yes |
+| bottom_temperature_target | Number | The target temperature of the bottom area. | Yes |
+| bottom_temperature_current | Number | The currently measured temperature of the bottom area. | Yes |
+| light_switch | Switch | Indicates if the light of the device is enabled. | No |
+| light_can_be_controlled | Switch | Indicates if the light of the device can be controlled. | Yes |
+| plate_is_present | Switch | Indicates if plate is present. | Yes |
+| plate_power_step | String | The power level of the heating plate. | Yes |
+| plate_power_step_raw | Number | The raw power level of the heating plate. | Yes |
+| door_state | Switch | Indicates if the door of the device is open. | Yes |
+| door_alarm | Switch | Indicates if the door alarm of the device is active. | Yes |
+| battery_level | Number | The battery level of the robotic vacuum cleaner. | Yes |
+
+### Coffee System
+
+| Channel ID | Channel Type ID |
+| ---------- | --------------- |
+| remote_control_can_be_started | remote_control_can_be_started |
+| remote_control_can_be_stopped | remote_control_can_be_stopped |
+| remote_control_can_be_switched_on | remote_control_can_be_switched_on |
```

Comment:
The Channel Type ID column seems to be redundant. Would a list be sufficient?

Reply:
You're right, it is redundant in almost all cases. The sole exception is the hob which uses multiple channels of the same type for the `plate_power_step` channels. So, better make a list for all other devices and leave the table for the hob?

Reply:
I think that'd be better. I'm wondering why the Channel Type ID is important to the user at all?

Reply:
In order to define the correct type of item in `.items` files the user needs to know the type of the channel which is defined by the channel type. Therefore, we included it in the documentation.

Reply:
Understood. You could add the Item type to the Channels in the lists which differ from above table.

Reply:
I changed everything to lists, the only special case is the hob where I used some additional text to explain.